spec/unit/provider/package/windows/exe_package_spec.rb

```diff
@@ -34,7 +34,6 @@
     {
       'DisplayName'      => ['My App', ''],
       'UninstallString'  => ['E:\uninstall.exe', ''],
-      'SystemComponent'  => [nil, 1],
       'WindowsInstaller' => [nil, 1],
       'ParentKeyName'    => [nil, 'Uber Product'],
       'Security Update'  => [nil, 'KB890830'],
```

Comment:
Any decision here? I think I'm also on the fence about not seeing security updates and hotfixes by default. 

It's too bad `package` doesn't have `query_options`